librosa/core/constantq.py

```diff
@@ -62,9 +63,9 @@ def cqt(y, sr=22050, hop_length=512, fmin=None, n_bins=84,
         Filter scale factor. Small values (<1) use shorter windows
         for improved time resolution.
 
-    aggregate : None or function
-        Aggregation function for time-oversampling energy aggregation.
-        By default, `np.mean`.  See `librosa.util.sync`.
+    aggregate : None or function [DEPRECATED]
```

Comment:
So aggregate does nothing now.  Should we even support it?  Won't code that references it be broken anyway?